assets/src/stories-editor/components/higher-order/with-keyboard-navigation-handler.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * WordPress dependencies
+ */
+import { withDispatch } from '@wordpress/data';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { UP, DOWN, RIGHT, LEFT, DELETE } from '@wordpress/keycodes';
+/**
+ * Internal dependencies
+ */
+import { ALLOWED_CHILD_BLOCKS, ALLOWED_MOVABLE_BLOCKS } from '../../constants';
+
+const applyWithDispatch = withDispatch( ( dispatch, props, { select } ) => {
+	const {	isReordering } = select( 'amp/story' );
+	const { getSelectedBlock } = select( 'core/block-editor' );
+	const { updateBlockAttributes, removeBlock } = dispatch( 'core/block-editor' );
+
+	const onKeyPress = ( event ) => {
+		const { keyCode, target } = event;
+		const { classList } = target;
+		const selectedBlock = getSelectedBlock();
+
+		if ( ! selectedBlock ) {
+			return;
+		}
+
+		if ( classList.contains( 'editor-rich-text__editable' ) && classList.contains( 'is-selected' ) ) {
```

Comment:
Isn't there a `isSelected` prop?

Reply:
Not that I can see 
```
attributes: {ordered: false, values: "", addedAttributes: 0, opacity: 100, positionTop: 30, …}
clientId: "b1540913-690e-4e23-8a0a-8f531c77301e"
innerBlocks: []
isValid: true
name: "core/list"
```

Reply:
Odd 🤔 

I thought it does:

https://github.com/WordPress/gutenberg/blob/29ad4d8740b04b5bf094b6901cccd71552767cb3/packages/block-editor/src/components/block-list/block.js#L459-L470

https://github.com/WordPress/gutenberg/blob/29ad4d8740b04b5bf094b6901cccd71552767cb3/packages/block-editor/src/components/block-edit/index.js#L34-L43

https://github.com/WordPress/gutenberg/blob/29ad4d8740b04b5bf094b6901cccd71552767cb3/packages/block-editor/src/components/block-edit/edit.js#L12-L34